plugins/modules/ini_file.py

```diff
@@ -471,16 +528,20 @@ def main():
         argument_spec=dict(
             path=dict(type='path', required=True, aliases=['dest']),
             section=dict(type='str'),
+            section_has=dict(type='list', elements='dict', options=dict(
+                option=dict(type='str'),
+                value=dict(type='str')
```

Comment:
Do I understand it correctly that you can also specify only one of `option` and `value`, or even none? I would expect that I have to specify both of them (i.e. both are required).

Reply:
I suppose they should both be required.. Can you do that by e.g. `option=dict(type='str', required=True)`?